### ISSUE TITLE
interactionCovariateIds in createCmAnalysis

### ISSUE THREAD

**Issue by chandryou** [OHDSI/CohortMethod#74]
Currently, cohortMethod supports interaction term analysis.
Can we generate cmAnalysisList.json by runCmAnalyses for interaction term analysis, too?

**Comment by schuemie** [comment]
Yes, as always the create...Args function mimic the single-study function arguments. So, for interaction terms you'll need to specify the [`interactionCovariateIds`](https://github.com/OHDSI/CohortMethod/blob/master/R/CreateArgFunctions.R#L401) argument in the `createFitOutcomeModelArgs` function.
